core/src/test/java/com/graphhopper/routing/weighting/custom/ExpressionBuilderTest.java

```diff
@@ -37,9 +37,8 @@ void setPriorityForRoadClass() {
         EncodingManager encodingManager = EncodingManager.create(encoder);
         CustomModel customModel = new CustomModel();
         customModel.getPriority().put("road_class == PRIMARY", 0.5);
-        // todo: can we get rid of this line here?
-        double maxSpeedFallback = customModel.getMaxSpeedFallback() == null ? encoder.getMaxSpeed() : customModel.getMaxSpeedFallback();
```

Comment:
Any special reason you included this code in test too?

Reply:
Yes this line is in `CustomWeighting#create` currently. I was hoping to be able to move it into `ExpressionBuilder#create` so I do not have to repeat it here.

These two lines:

```double maxSpeedTmp = customModel.getMaxSpeedFallback() == null ? baseFlagEncoder.getMaxSpeed() : customModel.getMaxSpeedFallback();
        SpeedAndAccessProvider speedAndAccessProvider = ExpressionBuilder.create(customModel, lookup, baseFlagEncoder.getMaxSpeed(), maxSpeedTmp, baseFlagEncoder.getAverageSpeedEnc());
```
look like redundant information is passed to `ExpressionBuilder#create`: maxSpeedTmp is just a combination of `customModel` and `baseFlagEncoder.getMaxSpeed()`. But we pass all three to ExpressionBuilder. I think I wanted to clean this up, but then we were not sure anymore if `maxSpeedFallback` is even needed (see other comment).